.github/workflows/build_and_test.yml

```diff
@@ -53,6 +58,12 @@ jobs:
       - name: Build Windows Client
         run: npm run action src/electron/package_windows
 
+      - name: Upload Windows Client
+        uses: actions/upload-artifact@v2
+        with:
+          name: unsigned-Outline-Client.exe
```

Comment:
Don't rename the file here. I meant changing the name we use in the build system. I don't want to see files changing names. Mutability is confusing. Please change the config to output the exact name the build system outputs. We can change the build output name in another PR.

Reply:
Ah, okay. We'll just have to track it manually then